python/ThirteenTeV/Hadronizer_ttWJets_TuneCP5_13TeV_MLM_5f_max2j_LHE_pythia8_cff.py

```diff
@@ -0,0 +1,39 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        processParameters = cms.vstring(
```

Comment:
Unless I am wrong, I don't see any specific settings associated to this dataset, isn't it the case?

Reply:
The only difference of the two is nJetMax, since it is one time +2jets and one time +1jet max

Reply:
I think is not worth having multiple, equal cfg whith the same name.
I would simply add 1 generic cfg (i.e. removing the process in the cfg name) whenever needed
i.e, check if your cfg is already existing here
https://github.com/Andrej-CMS/genproductions/tree/master/python/ThirteenTeV/Hadronizer
also, why
'JetMatching:nJetMax = 5' fot ttW and 'JetMatching:nJetMax = 4', for ttZ ?

Reply:
@perrozzi 

I will then remove both of the hadronizer settings, since they are generic for MLM matching. As far as I understand, nJetMax is the number of  partons in the born ME: tt+W+2jets=5, tt+Z+1Jet=4. 

Have I misunderstood something?

Reply:
@kdlong @qliphy can you please crosscheck?